styled/checkout.js

```diff
@@ -65,44 +29,16 @@ export const LineItem = styled.View`
   height: 88px;
```

Comment:
Sorry I might've originally suggested this, but if you remove this height, the Qty line wouldn't look so smushed.